website/content/en/docs/building-operators/golang/references/multi-group.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Support to Multi-Group APIs
```

Comment:
I'd rather link the corresponding [kubebuilder doc](https://kubebuilder.io/migration/multi-group.html) than write our own.

Reply:
agreed